### ISSUE TITLE
Incorrect response for APIs to list VMI Presets and storageclasses

### ISSUE THREAD

**Issue by sankalp-r** [kubermatic/kubermatic#8719]
### What happened?
1. APIs to list VMI preset(`providers/kubevirt/vmflavors`) and storage classes(`providers/kubevirt/storagrclasses`) for KubeVirt provider with `Credential` as header gives incorrect response:
`{
    "error": {
        "code": 500,
        "message": "illegal base64 data at input byte 10"
    }
}`

2. APIs to list VMI preset and storage classes for KubeVirt provider without credentials gives incorrect response:
`{
    "error": {
        "code": 500,
        "message": "illegal base64 data at input byte 10"
    }
}`

### What should happen?
API endpoints should return correct response with list of VMI presets or storageclasses.

### How to reproduce?

Use rest-client to fire the API with appropriate headers/parameters, for example:

```bash
curl --location --request GET '<kkp-url>/api/v2/providers/kubevirt/vmflavors' \
--header 'accept: application/json, text/plain, */*' \
--header 'Credential: kubermatic' \
--header 'Authorization: Bearer <jwt_token>'
```

### Environment

- KKP Version: v2.19.0-alpha.5
- Shared or separate master/seed clusters?: shared
- Master/seed cluster provider: AWS

**Comment by sankalp-r** [comment]
Reference: #8716

**Comment by hdurand0710** [comment]
A change was requested in the PR https://github.com/kubermatic/kubermatic/pull/8441/files/64228cf1b6cdab668390d078a609d9e3fd6e3a58 to only have base64 encoded kubeconfig.

We need to change the kubevirt provider preset to contain a base64 encoded preset (on QA but also on dev).

**Comment by hdurand0710** [comment]
@sankalp-r : will be fixed in next KKP release (v2.19.0-alpha.7) -> back to you for testing Monday for the credentials from secret.
